lutris/runners/libretro.py

```diff
@@ -45,6 +45,11 @@
     ("MAME (Arcade)", "mame", "Arcade"),
     ("Mednafen GBA (Game Boy Advance)", "mednafen_gba", "Nintendo Game Boy Advance"),
     ("Mednafen NGP (SNK Neo Geo Pocket)", "mednafen_ngp", "SNK Neo Geo Pocket"),
+    (
+        "Mednafen PCE (TurboGrafx-16)",
+        "mednafen_pce",
+        "NEC PC Engine (TurboGrafx-16)",
```

Comment:
Got an extra comma at the end here. While it doesn't really effect it, it might be good to match the format of the rest of the entries.

Reply:
@RobLoach Fixed it and Fixed the formatting to be uniform across the core entries